deploy/csi-operator.yaml

```diff
@@ -962,8 +977,12 @@ spec:
               # needed so that any mounts setup inside this container are
               # propagated back to the host machine.
               mountPropagation: "Bidirectional"
-            - name: iscsiadm-bin
+            - name: host-root
+              mountPath: /host
+              mountPropagation: "HostToContainer"
+            - name: chroot-iscsiadm
               mountPath: /sbin/iscsiadm
+              subPath: iscsiadm
```

Comment:
Q: Why we need subpath?

Reply:
because we don't want to mount as volume , i.e. to not change the path inside the container

Reply:
This also ensures that any symlinks get resolved to inside the chroot environment.